.docker/build/config.json

```diff
@@ -4,7 +4,7 @@
     "BaseContainerIsEverestImage" : true,
     "BaseContainerImageName" : "fstar",
     "BaseContainerImageTagOrCommitId": "latest",
-    "BranchName" : "master",
+    "BranchName" : "taramana_steel_ptr",
```

Comment:
This change is for CI only. It should not go to `master`. However, reverting it requires F* `steel` branch to be merged to `master` first.